src/ros/build-env-utils.ts

```diff
@@ -68,23 +68,27 @@ async function updateCppPropertiesInternal(): Promise<void> {
         });
     }));
 
-    let platform: string = os.platform();
-    let c_cpp_properties: any = {
+    if (process.platform === "win32") {
+        includes.push(path.join("C:", "opt", "rosdeps", "x64", "include", "**"));
+    } else {
+        includes.push(path.join("/", "usr", "include"));
+    }
+
+    // https://github.com/Microsoft/vscode-cpptools/blob/master/Documentation/LanguageServer/c_cpp_properties.json.md
+    const cppProperties: any = {
```

Comment:
`any` is unsafe and should not be used, I think such an configuration interface could be retrieved from the `vscode-cpptools` npm package. However, to actually integrate that into this extension could take time. Do you think we should do that in this change?

Reply:
Integrating `vscode-cpptools` sounds like the right way to go. Just curious how much we need to add to integrate with `vscode-cpptools`?